ISISComputingGroup/EPICS-ioc
# Ticket 4303: Integrate CRISP jaws

### Description of work

Added an IOC for the motion support of tcIOC.

### To test

See https://github.com/ISISComputingGroup/IBEX/issues/4303 and main PR at https://github.com/ISISComputingGroup/EPICS-tcIoc/pull/4

Ensure that you test autosave remembers all relevant motion parameters.
The PV prefix currently does not contain the IOC name as it will violate PV name lengths. This means only one of these can be run per instrument. This is currently not an issue but ticket is at https://github.com/ISISComputingGroup/IBEX/issues/4669

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
